example/fomo-example-autoload.php

```diff
@@ -1,28 +1,25 @@
 <?php
 
-$loader = include 'vendor/autoload.php';
-$loader->add('Fomo\Fomo', __DIR__.'/src');
-
-// activate the autoloader
-$loader->register();
+require_once dirname(__DIR__) . '/vendor/autoload.php';
 
 $token = getenv('FOMO_TOKEN');
 
-if (!$token) {
-    echo PHP_EOL . PHP_EOL . "Please setup the env variable FOMO_TOKEN=<YOUR_FOMO_TOKEN_HERE>". PHP_EOL . PHP_EOL;
+if (! $token) {
```

Comment:
@mjaschen can you point me to docs where this is a better code style? i'm more a Ruby guy.

Reply:
@mjaschen going to close this in a day or so, otherwise happy to merge if you can explain a bit further. also perhaps check out latest updates.

Reply:
Hi @ryanckulp What exactly do you want a more detailed explanation for? (In case it's the inserted space after the NOT operator: that's common in many PHP projects, for readability reasons. Feel free to remove it :-))